src/extra/AutoUI/Filters/PersistentFilters.tsx

```diff
@@ -44,27 +46,24 @@ const isQueryStringFilterRuleset = (
 	!!rule?.length &&
 	rule?.every(isListQueryStringFilterRule);
 
-const listFilterQuery = (schema: JSONSchema, rules: JSONSchema[]) => {
-	const queryStringFilters = rules.map((filter) => {
-		// TODO: The typings need to be fixed in rendition.
-		const flatSchema = flattenSchema(schema);
-		const flattenFilter = flattenSchema(filter);
-		const signatures = decodeFilter(
-			flatSchema,
-			flattenFilter,
-		) as FilterSignature[];
+export const listFilterQuery = (filters: JSONSchema[]) => {
+	const queryStringFilters = filters.map((filter) => {
+		const signatures = getSignatures(filter) as FilterSignature[];
 		return signatures.map<ListQueryStringFilterObject>(
-			({ field, operator, value }) => ({
+			({ key, title, field, operator, value, refScheme }) => ({
+				k: key,
+				t: title,
 				n: field,
 				o: operator,
 				v: value,
+				r: refScheme,
 			}),
 		);
 	});
 	return qs.stringify(queryStringFilters);
 };
 
-const loadRulesFromUrl = (
+export const loadRulesFromUrl = (
```

Comment:
Howcome we are exporting this now? We are not importing it anywhere

Reply:
Good catch, we re-write it in the UI, but I then decided to export all the persistent filters, so we can remove all that code from the dashboard as well. :pray: